system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -465,16 +464,17 @@ class CLdapConfig : implements ILdapConfig, public CInterface
 
             for(int retries = 0; retries <= LDAPSEC_MAX_RETRIES; retries++)
             {
+                getLdapHost(hostbuf);
```

Comment:
Why move this line?  It now will read it each time round the retry loop?

Reply:
That's the point, and the problem this Jira is fixing. A different thread may blacklist the currently selected host, in which case we always need to get the latest

Reply:
That needs a comment in the code because it is not clear from the code - especially given the way the outer loop is structured.